ext/standard/var.c

```diff
@@ -634,7 +663,9 @@ PHPAPI void php_var_export(zval *struc, int level) /* {{{ */
 	smart_str buf = {0};
 	php_var_export_ex(struc, level, &buf);
 	smart_str_0(&buf);
-	PHPWRITE(ZSTR_VAL(buf.s), ZSTR_LEN(buf.s));
+	if (EG(exception)) {
```

Comment:
I don't understand this check, isn't this supposed to be `!EG(exception)` ?

Reply:
Ofc! Thanks for catching this. The function is unused, but might still be used in extensions. It should use the result of `php_var_export_ex()` anyway.